Error: Cannot find module '@cdktf/provider-aws'

I ran `wing compile hello.w` using the latest `@monadahq/wing` with:

```
// hello.w
bring cloud;
```

And got

```
Error: Cannot find module '@cdktf/provider-aws'
```

It looks like the issue is `@monadahq/wing` takes `@monadahq/wingsdk` as a bundled dependency, but `@monadahq/wingsdk` has several peer dependencies so they aren't automatically bundled.

--- a/apps/wing/package.json
+++ b/apps/wing/package.json
@@ -20,6 +20,10 @@
   },
   "dependencies": {
     "@monadahq/wingsdk": "file:../../libs/wingsdk",
+    "@cdktf/provider-aws": "*",
+    "@monadahq/polycons": "*",
+    "cdktf": "*",
+    "constructs": "*",
     "@types/node": "^18.7.22",
     "commander": "^9.4.0"
   },
@@ -29,6 +33,10 @@
     "@types/node": "^18.7.22"
   },
   "bundledDependencies": [
-    "@monadahq/wingsdk"
+    "@monadahq/wingsdk",
+    "@cdktf/provider-aws",
+    "@monadahq/polycons",
+    "cdktf",
+    "constructs"
   ]
 }

